packages/wallet/backend/src/rapyd/controller.ts

```diff
@@ -182,4 +182,40 @@ export class RapydController implements IRapydController {
       next(e)
     }
   }
+
+  public getDailyRate = async (
+    req: Request,
+    res: CustomResponse<DailyRateResponse>,
+    next: NextFunction
+  ) => {
+    try {
+      const {
```

Comment:
I think we should add a schema in validation.ts for this request as well, and you'd be able to remove the extra checks and throws further down